packages/vite/src/client.ts

```diff
@@ -48,15 +51,17 @@ export async function buildClient (ctx: ViteBuildContext) {
       'import.meta.browser': true,
       'import.meta.nitro': false,
       'import.meta.prerender': false,
-      'module.hot': false
+      'module.hot': false,
+      'global': 'globalThis'
     },
     optimizeDeps: {
       entries: [ctx.entry]
     },
     resolve: {
       alias: {
         '#build/plugins': resolve(ctx.nuxt.options.buildDir, 'plugins/client'),
-        '#internal/nitro': resolve(ctx.nuxt.options.buildDir, 'nitro.client.mjs')
+        '#internal/nitro': resolve(ctx.nuxt.options.buildDir, 'nitro.client.mjs'),
+        ..._env.alias
```

Comment:
The code here overrides user config so think we should add these subsequently via `defu` to allow the user to override these settings, granularly.

Reply:
Good catch! For now we only have this option so added simple override to periororitze user conf (besides, it will be configurable also using `unenv: {}` option for alias